tool.go

```diff
@@ -266,6 +258,13 @@ func main() {
 		Use:   "run [gofiles...] [arguments...]",
 		Short: "compile and run Go program",
 	}
+	cmdRun.Flags().AddFlag(flagVerbose)
```

Comment:
> The only change that I see being possible is to create all flags within a dummy flag set, then look them up, and add to individual commands.

I've done that in fe2b2c69b8d23430b4a6c8240ad44aa0b15fddc7, PTAL.

Let me know if you have ideas for how to improve it further. I don't see any other ways.